### ISSUE TITLE
The default tuple element names config should align with the new pascal case guidance

### ISSUE THREAD

**Issue by jnm2** [DotNetAnalyzers/StyleCopAnalyzers#3019]
The BCL and language teams have decided on pascal case for tuple element names and have added an API to the BCL which sets the precedent. (https://github.com/dotnet/corefx/pull/35595, 
https://github.com/dotnet/corefx/issues/33553)

Under https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1316.md#rule-description:

> For example, the following code would produce a violation of this rule (with default settings):
> 
> ```csharp
> public class ExampleClass
> {
>   public (int ValueA, int ValueB) ExampleMethod() // SA1316
>   {
>   }
> }
> ```

This should not default to being a violation. I would even expect that camel case would be considered a violation as soon as this rule is enabled in a new project.

This rule does not exist in the latest release version, so maybe backwards compatibility won't be an issue.

**Comment by vweijsters** [comment]
I'll change the default to pascal case.

**Comment by sharwell** [comment]
The casing for SA1316 predates the framework design guidelines, and wasn't part of that discussion. I would prefer to keep SA1316 in its current form, or _at minimum_ make it a configuration option with camelCase as the default.

**Comment by sharwell** [comment]
I'd also settle for making it an option with the default as PascalCase. It's not the configuration I would use but would have less impact on some users trying to move to 1.2.

**Comment by vweijsters** [comment]
The current implementation already supports an option, I just changed the default from camelCase to PascalCase.

**Comment by kofifus** [comment]
As far as I know there is no official decision on this. The [docs](https://docs.microsoft.com/en-us/dotnet/csharp/tuples use camelCase

**Comment by jnm2** [comment]
@kofifus The official decision is confirmed at https://github.com/dotnet/corefx/issues/33553#issuecomment-531420515 which mentions that they are working on docs.

**Comment by jnm2** [comment]
It's probably also worth mentioning that the C# and framework design teams came from opposite points of view initially, so it's not surprising that the C#-specific docs used camel case.
